k8s/overlays/azure/test/kustomization.yaml

```diff
@@ -77,6 +79,14 @@ patches:
               - name: cse-import-d2cc-runner-itools-configmap-volume
                 configMap:
                   name: cse-runner-itools-azure-test-configmap
+  # Scale-up to 4 instances and adapt CSE runner itools config
+  - patch: |-
+      kind: Deployment
+      apiVersion: apps/v1
+      metadata:
+        name: swe-d2cc-runner
+      spec:
+        replicas: 4
```

Comment:
Could you put replicas to 1 until the bug of parallelization is fixed ?

Reply:
It is fixed